tests/php/test-class-amp-schema-org-metadata.php

```diff
@@ -69,8 +80,6 @@ public function test_transform( $json, $expected ) {
 	 * @covers ::transform
 	 */
 	public function test_empty_metadata_configuration() {
-		add_filter( 'amp_schemaorg_metadata', '__return_empty_array' );
```

Comment:
The corresponding `remove_filter()` needs to be removed as well.

Reply:
Removed in 15310db2a. It wasn't needed in the first place since filters get reset when tests are torn down.